### ISSUE TITLE
excluded urls are not excluded from alternate entry

### ISSUE THREAD

**Issue by FranckFreiburger** [nuxt-modules/sitemap#273]
nuxt configuration:
  i18n: 3 langs: en, es, fr (with default en)

  sitemap:
```
	sitemap: {
		xsl: false,
		exclude: [
			'/en/accrochageCertificateur',
			'/es/accrochageCertificateur',
		],
```

accrochageCertificateur page is properly excluded from `/en-sitemap.xml` and `/es-sitemap.xml` but is still present as `alternate` in `/fr-sitemap.xml` :

```xml
...
<loc>http://localhost:3000/fr/accrochageCertificateur</loc>
    <xhtml:link rel="alternate" hreflang="en" href="http://localhost:3000/en/accrochageCertificateur" />
    <xhtml:link rel="alternate" hreflang="es" href="http://localhost:3000/es/accrochageCertificateur" />
    <xhtml:link rel="alternate" hreflang="fr" href="http://localhost:3000/fr/accrochageCertificateur" />
    <xhtml:link rel="alternate" hreflang="x-default" href="http://localhost:3000/en/accrochageCertificateur" />
</url>
...
```

**Comment by harlan-zw** [comment]
Thanks, this should be fixed in [v5.1.5](https://github.com/nuxt-modules/sitemap/releases/tag/v5.1.5).

**Comment by FranckFreiburger** [comment]
hello, can you confirm/reopen this issue ?
